world-server/src/world/World.ts

```diff
@@ -10,9 +10,12 @@ import { Router } from 'world-core';
 import random = require('random');
 import seedrandom = require('seedrandom');
 import SimplexNoise = require('simplex-noise');
+import { ChatHandler } from "./chat/ChatHandlers";
+import { router } from "websocket";
```

Comment:
Where is this used?

Reply:
`router` is not used, removed. The chat handler is registered in the `World constructor` function.